.gitlab-ci.yml

```diff
@@ -208,6 +212,23 @@ secrets:
     - *common_ci_setup
   script:
     - tox -e secrets
+  only:
```

Comment:
I think this is not part of your PR, but does this mean that mypy stage only runs during the scheduled job? Should we remove this?

Reply:
Whoops, I see now it's the secrets stage. Nevermind.